site/src/modules/management/SidebarView.tsx

```diff
@@ -43,25 +41,29 @@ export const SidebarView: FC<SidebarProps> = ({
 	activeOrganizationName,
 	organizations,
 	permissions,
-	experiments,
 }) => {
+	const { showOrganizations } = useDashboard();
+
 	// TODO: Do something nice to scroll to the active org.
 	return (
 		<BaseSidebar>
-			<header>
-				<h2 css={styles.sidebarHeader}>Deployment</h2>
-			</header>
+			{showOrganizations && (
```

Comment:
Why would showOrganizations need to be true to show only the Deployment header? Is there some reason to show DeploymentSettingsNavigation without the Header text?

Reply:
it just looks weird for it to say deployment twice without the organizations section below. it's weird, but we'll get rid of this when we adopt the designs we went over with Christin.